FSAE.X/CAN.h

```diff
@@ -11,8 +11,7 @@
  * CAN message IDs
  */
 #define ERROR_ID        0x00F
-#define ANALOG_REAR_ID  0x050
-#define ANALOG_FRONT_ID 0x060
+#define SPM_ID          0x050
```

Comment:
For sure.  I didn't realize that the ID of the message had to do with its priority at all.